project/CentralRequirements.scala

```diff
@@ -48,7 +48,7 @@ object CentralRequirementsPlugin extends AutoPlugin {
       if (isSnapshot.value)
         Some("snapshots" at nexus + "content/repositories/snapshots")
```

Comment:
I don't know whether it makes sense to update this as well, since we don't execute the sonatypeRelease step on snapshots.

Reply:
I guess we can simplify further:

> If isSnapshot.value is true (e.g., if the version name contains -SNAPSHOT), publishSigned task will upload files to the Sonatype Snapshots repository without using the local bundle folder.